runtime/src/accounts_db.rs

```diff
@@ -4443,16 +4443,20 @@ impl AccountsDb {
             // randomly shrink ancient slots
             // this exercises the ancient shrink code more often
             let written_bytes = storage.written_bytes();
+            let mut alive_ratio = 0;
```

Comment:
Calculating `alive_ratio` is relatively cheap - atomic load, multiply, divide.
Seems preferable to me to just calculate it here and log the ratio regardless of randomness, since we're also logging whether it was a random shrink or not.